OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -86,8 +86,8 @@ public abstract class ScriptGlobal : ScriptObjectWrapper
 		public ScriptGlobal(ScriptContext context)
 			: base(context)
 		{
-			// The 'this.' resolves the actual (subclass) type
-			var type = this.GetType();
+			// GetType resolves the actual (subclass) type
+			var type = GetType();
```

Comment:
Are you confident that is true across the various older mono releases?
This was definitely an issue at the time, otherwise I wouldn't have left a comment specifically mentioning it.